app/routes/__index/domains/index.tsx

```diff
@@ -51,6 +51,12 @@ export const action = async ({ request }: ActionArgs) => {
     });
   }
 
+  if (record.status !== 'active') {
+    throw new Response('Record is not active, action forbiden', {
+      status: 403,
+    });
+  }
```

Comment:
I was wondering... if we are going to redirect the user back into domain when they are trying to navigate the page, where does the `throw` happense?

Reply:
We are redirecting users back only from the domain edit page, while if they try to do something with the inactive record on domains list page we throw an error